### PR TITLE
Create todo implementation with Jotai

### PR DESCRIPTION
**Pull request by divisandeep94**

### CODE REVIEW

File: app/components/create-todo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import { appLabels } from '../../constants'
+import { todoListAtom } from '../list-item'
+import { v4 as uuid } from 'uuid'
+
+const CreateTodo = () => {
+  const router = useRouter()
+  const [todoValue, setTodoValue] = useState('')
+  const [todoList, setTodoList] = useAtom(todoListAtom)
+
+  const handleNewTodoCancel = () => {
```
Comment (vaishali3012) [inline app/components/create-todo/index.tsx]: We can directly say handleCancel and handleCreate as className is already create-todo so better not to repeat

File: app/components/list-item/index.tsx [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
-}
+export const todoListAtom = atom<IListItemType[]>([])
+
+const ListItem = () => {
+  const [todoList, setTodoList] = useAtom(todoListAtom)
+
+  const handleTodoStatusChange = (itemId: string) => {
+    const updatedTodos = todoList.map((todo) => {
+      if (todo.id === itemId) {
+        if (todo.status === 'Done') todo.status = 'Active'
```
Comment (vaishali3012) [inline app/components/list-item/index.tsx]: Try to prefer ternary operator for single if else

File: app/components/list-item/index.tsx [hunk 3]
```diff
@@ -1,26 +1,54 @@
...
+                id={todo.id}
+                className="cursor-pointer peer"
+                onChange={() => {
+                  handleTodoStatusChange(todo.id)
+                }}
+              />
+              <label htmlFor={todo.id} className="cursor-pointer  peer-checked:text-slate-500">
+                {todo.name}
               </label>
+              {todo.status === 'Active' && (
```
Comment (vaishali3012) [inline app/components/list-item/index.tsx:45]: We can use todo.status === 'Active' one time only for both edit and delete

File: app/components/navbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+'use client'
```
Comment (vaishali3012) [inline app/components/navbar/index.tsx]: Do we need useClient here

**Review by vaishali3012 — CHANGES_REQUESTED** [review]

File: app/components/navbar/index.tsx (hunk 4 above)
Comment (divisandeep94) [reply app/components/navbar/index.tsx]: Removed use client.

File: app/components/list-item/index.tsx (hunk 3 above)
Comment (divisandeep94) [reply app/components/list-item/index.tsx:45]: Wrapped both edit and delete under single condition.

File: app/components/list-item/index.tsx (hunk 2 above)
Comment (divisandeep94) [reply app/components/list-item/index.tsx]: Updated the nested if else by using ternary operator.

File: app/components/create-todo/index.tsx (hunk 1 above)
Comment (divisandeep94) [reply app/components/create-todo/index.tsx]: Updated both the function names.

**Review by vaishali3012 — APPROVED** [review]

**Pull request merged**